packages/@aws-cdk/aws-events/test/test.rule.ts

```diff
@@ -607,11 +607,7 @@ export = {
           region: 'us-west-2',
         },
       });
-      const rule = new Rule(sourceStack, 'Rule', {
-        eventPattern: {
-          source: ['some-event'],
-        },
-      });
+      const rule = new Rule(sourceStack, 'Rule');
```

Comment:
Is this a regression in behavior? Why was this needed to change? I’d expect this test to remain in tact. If there’s another case you are trying to cover, please add an additional test.

Reply:
The test was not written in an exhaustive enough way before - in this new form, it fails without the production changes, even though the assertions stayed the same.

Reply:
Not sure what that means... as far as I understand the previous code should work and therefore the old test is relevant. If it now fails, then something is wrong. If you believe we need additional coverage, please add an additional test.

Reply:
Added a new test.